src/org/usfirst/frc/team2335/robot/subsystems/Drive.java

```diff
@@ -1,12 +1,32 @@
 package org.usfirst.frc.team2335.robot.subsystems;
 
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Victor;
 import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 
 public class Drive extends Subsystem
 {
-	//TODO: Add motor controllers
-
-	//TODO: Add drive commands
+	Victor leftMotor, rightMotor;
+	
+	DifferentialDrive drive;
+		
+	public Drive()
+	{
+		//Motor controller definitions
+		leftMotor = new Victor(Robot.LEFT_MOTOR);
+		rightMotor = new Victor(Robot.RIGHT_MOTOR);
+	
+		
+		//Drive controller definition
+		drive = new DifferentialDrive(leftMotor, rightMotor);
+	}
+	
+	public void drive(double moveVal, double rotateVal)
+	{			
+		drive.arcadeDrive(moveVal, rotateVal);
+	}
 	
     public void initDefaultCommand()
```

Comment:
Can we get rid of this?

Reply:
No, because we don't want to access an objects methods from another class, it's not safe.